src/test/regress/expected/shard_rebalancer.out

```diff
@@ -1726,13 +1767,6 @@ SELECT citus_add_rebalance_strategy(
         0
     );
 ERROR:  signature for shard_cost_function is incorrect
-DETAIL:  number of arguments of shard_cost_no_arguments should be 1, not 0
```

Comment:
The `\set VERBOSITY terse` that was added here removed this DETAIL. It's the only place where we test this so I think it's nice to show it here. Can you create a separate PR where you set the VERBOSITY higher again?

Reply:
> g_dist_rebalance_strategy_trigger_func() line 5 at PERFORM

The line numbers were not consistent, it might be due to macos or something. I'll retry

Reply:
Hmm okay, I guess the CONTEXT part can be removed from the tests if it causes issues. But the detail bit is nice to have there.